shell/platform/fuchsia/flutter/tests/integration/mouse-input/mouse-input-view/lib/mouse-input-view.dart

```diff
@@ -108,20 +108,19 @@ class MyApp {
       double wheelXPhysicalPixel,
       double wheelYPhysicalPixel}) {
     print('mouse-input-view reporting mouse input to MouseInputListener');
-    final message = utf8.encoder
-        .convert(json.encode({
-          'method': 'MouseInputListener.ReportMouseInput',
-          'local_x': localX,
-          'local_y': localY,
-          'time_received': timeReceived,
-          'component_name': 'touch-input-view',
-          'buttons': buttons,
-          'phase': 'asdf',
-          'wheel_x_physical_pixel': wheelXPhysicalPixel,
-          'wheel_y_physical_pixel': wheelYPhysicalPixel,
-        }))
-        .buffer
-        .asByteData();
+    final message = utf8.encode(json.encode({
+        'method': 'MouseInputListener.ReportMouseInput',
+        'local_x': localX,
+        'local_y': localY,
+        'time_received': timeReceived,
+        'component_name': 'touch-input-view',
+        'buttons': buttons,
+        'phase': 'asdf',
+        'wheel_x_physical_pixel': wheelXPhysicalPixel,
+        'wheel_y_physical_pixel': wheelYPhysicalPixel,
+      }))
+      .buffer
+      .asByteData();
```

Comment:
Make this `final message = ByteData.sublistView(utf8.encode(json.encode({...})));` like the change in `parent_view.dart`?

Reply:
Thanks, missed that one.